mobile/src/main/java/ch/epfl/sdp/mobile/application/tournaments/TournamentFacade.kt

```diff
@@ -63,6 +63,7 @@ class TournamentFacade(private val auth: Auth, private val store: Store) {
       user: AuthenticatedUser,
       name: String,
       maxPlayers: Int,
+      creationTime: Long,
```

Comment:
It's probably better to "hide" the creationTime from the caller and let the `TournamentFacade` populate it, since creating a tournament should not be done in the future or in the past.